### DISCUSSION TITLE
Show/hide videos that are 1 minute or less #ShortsBlock

### DISCUSSION THREAD

**Discussion by pieterhouwen** [ReVanced/revanced-discussions#143] [Question]
```printf "Hello World!\n"```

I don't like to watch shorts or have them come up in my feed, can you implement a way to hide these shorts from the YT home screen and channel listings?

Because shorts are so easy to make they really pollute the Channel Videos section of any YT channel.

For example: https://www.youtube.com/c/LegalEagle/videos

Many thanks.

```exit 0```

**Comment by NotHuza** [comment]
And put way to hide shorts button or make way to resplase this button

**Comment by tywentghxst** [comment]
Completely agree, this is needed to actually view the content we want, yt has become over saturated and aggressively pushing shorts Dow our throats because TikTok

**Comment by TheJeterLP** [comment]
Just use hide-shorts-button patch
